config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
> Make transparent where things came from.

I don't see how using a prefix affects that

> Make sure there are no accidental clashes between the environment the server sets, and the environment you set in your env file.

There would need to be at least one hypothetical clash for doing that to make sense IMO. Using a prefix prevents doing things like this:

    $ DATABASE_URL=xxx; bin/cake
    $ DATABASE_URL=xxx; bin/foo.py
    $ DATABASE_URL=xxx; bin/foo.rb

I don't see a need or benefit to using a prefix - it just complicates things for the default case, to account for an edge case scenario (which could instead use different env vars, or remove them etc.).